app/main.py

```diff
@@ -2,0 +1,15 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
The use of `split()` is correct here, and it is called only once, which aligns with the checklist requirement to call each necessary method only once.